src/vivarium_public_health/dataset_manager/hdf.py

```diff
@@ -15,25 +15,37 @@
 
 
 def touch(path: str, append: bool):
-    """Creates an hdf file if necessary or errors if trying to append to a non-existent file.
+    """Creates an hdf file or wipes an existing file if necessary.
+
+    If the file exists and append is not true, the existing file
+    will be destroyed. If the file exists and append is true, touch
+    will do nothing.
 
     Parameters
     ----------
     path :
         The string path to the hdf file.
     append :
-        Whether or not we want to append to an existing file.
+        Whether an existing artifact should be preserved and appended to
 
     Raises
     ------
     FileNotFoundError
         If attempting to append to a non-existent file."""
+
     path = Path(path)
 
-    if append and not path.is_file():
-        raise FileNotFoundError("You indicated you want to append to an existing artifact "
-                                f"at {path} but no such artifact exists.")
-    elif not append:
+    if path.is_file() and append:
+        pass
+    if path.is_file() and not append:
+        with tables.open_file(str(path), mode='w'):
+            pass
+    elif not path.is_file() and append:
+        raise FileNotFoundError("You indicated you want to append to "
+                                f"an existing artifact at {path} but no "
+                                "such artifact exists. remove --append if "
```

Comment:
I'm not familiar enough with how this gets used to know, but is it only ever called by your build_artifact script? If not, the reference to --append might be confusing

Reply:
Well, the append comes here from the config which is set in the build_artifact script, and you can hit this error as a result of calling build_artifact (this is the spot where I was getting the original error that started these PRs)

Reply:
But, I don't know if touch would be called somewhere else that would then make this message confusing

Reply:
Easiest would just be to remove the extra sentence. We could catch this exception in build artifact and add the message about the append flag, I suppose

Reply:
Yeah I guess saying "you indicated you want to append" does make it clear in the context of build_artifact that you're referring to the --append flag and it's more general